gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -249,7 +249,7 @@ const postChangeUnissuedFacility = async (req, res) => {
  * post for changing unissued facilities from application preview
  * redirects to application preview once changed
  * @param {req}
- * @returns {res}
+ * @param {res}
```

Comment:
`res` and `req` should not be in the `{}` - my guess is that this should be `@param {import('express').Request} req` and `@param {import('express').Response} res`

Reply:
Lets keep this for now and ill address in another PR tomorrow